api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -3412,5 +3412,5 @@ public void shouldSetTheCorrectSortWeightWhenAddingAnOrderWithANegativePosition(
 		expectedException.expect(APIException.class);
 		expectedException.expectMessage("Cannot add a member which is out of range of the list");
 		secondSavedOrderGroup.addOrder(newOrderWithInvalidPosition, secondSavedOrderGroup.getOrders().size() + 1);
-	}
```

Comment:
Can you revert this format change in this file before I can merge the code please? Sorry! I should have seen this earlier.